pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -575,3 +576,284 @@ func TestValidateKubeSchedulerConfigurationV1beta3(t *testing.T) {
+	metricsBindAddrInvalid := validConfig.DeepCopy()
+	metricsBindAddrInvalid.MetricsBindAddress = "0.0.0.0:9090"
+
+	healthzBindAddrInvalid := validConfig.DeepCopy()
+	healthzBindAddrInvalid.HealthzBindAddress = "0.0.0.0:9090"
+
+	percentageOfNodesToScore101 := validConfig.DeepCopy()
+	percentageOfNodesToScore101.PercentageOfNodesToScore = int32(101)
+
+	schedulerNameNotSet := validConfig.DeepCopy()
+	schedulerNameNotSet.Profiles[1].SchedulerName = ""
+
+	repeatedSchedulerName := validConfig.DeepCopy()
+	repeatedSchedulerName.Profiles[0].SchedulerName = "other"
+
+	differentQueueSort := validConfig.DeepCopy()
+	differentQueueSort.Profiles[1].Plugins.QueueSort.Enabled[0].Name = "AnotherSort"
+
+	oneEmptyQueueSort := validConfig.DeepCopy()
+	oneEmptyQueueSort.Profiles[0].Plugins = nil
+
+	extenderNegativeWeight := validConfig.DeepCopy()
+	extenderNegativeWeight.Extenders[0].Weight = -1
+
+	invalidNodePercentage := validConfig.DeepCopy()
+	invalidNodePercentage.Profiles[0].PluginConfig = []config.PluginConfig{
+		{
+			Name: "DefaultPreemption",
+			Args: &config.DefaultPreemptionArgs{MinCandidateNodesPercentage: 200, MinCandidateNodesAbsolute: 100},
+		},
+	}
+
+	invalidPluginArgs := validConfig.DeepCopy()
+	invalidPluginArgs.Profiles[0].PluginConfig = []config.PluginConfig{
+		{
+			Name: "DefaultPreemption",
+			Args: &config.InterPodAffinityArgs{},
+		},
+	}
+
+	duplicatedPluginConfig := validConfig.DeepCopy()
+	duplicatedPluginConfig.Profiles[0].PluginConfig = []config.PluginConfig{
+		{
+			Name: "config",
+		},
+		{
+			Name: "config",
+		},
+	}
+
+	mismatchQueueSort := validConfig.DeepCopy()
+	mismatchQueueSort.Profiles = []config.KubeSchedulerProfile{
+		{
+			SchedulerName: "me",
+			Plugins: &config.Plugins{
+				QueueSort: config.PluginSet{
+					Enabled: []config.Plugin{{Name: "PrioritySort"}},
+				},
+			},
+			PluginConfig: []config.PluginConfig{
+				{
+					Name: "PrioritySort",
+				},
+			},
+		},
+		{
+			SchedulerName: "other",
+			Plugins: &config.Plugins{
+				QueueSort: config.PluginSet{
+					Enabled: []config.Plugin{{Name: "CustomSort"}},
+				},
+			},
+			PluginConfig: []config.PluginConfig{
+				{
+					Name: "CustomSort",
+				},
+			},
+		},
+	}
+
+	extenderDuplicateManagedResource := validConfig.DeepCopy()
+	extenderDuplicateManagedResource.Extenders[0].ManagedResources = []config.ExtenderManagedResource{
+		{Name: "foo", IgnoredByScheduler: false},
+		{Name: "foo", IgnoredByScheduler: false},
+	}
+
+	extenderDuplicateBind := validConfig.DeepCopy()
+	extenderDuplicateBind.Extenders[0].BindVerb = "foo"
+	extenderDuplicateBind.Extenders = append(extenderDuplicateBind.Extenders, config.Extender{
+		PrioritizeVerb: "prioritize",
+		BindVerb:       "bar",
+	})
+
+	goodInvalidPlugins := validConfig.DeepCopy()
+	goodInvalidPlugins.Profiles[0].Plugins.Score.Enabled = append(goodInvalidPlugins.Profiles[0].Plugins.Score.Enabled, config.Plugin{Name: "PodTopologySpread", Weight: 2})
+
+	badInvalidPlugins := validConfig.DeepCopy()
```

Comment:
```suggestion
	invalidPlugins := validConfig.DeepCopy()
```

Reply:
It's a comparison to `goodInvalidPlugins`.